governance.md

```diff
@@ -18,88 +36,191 @@ The committee MUST:
+- Hold staggered terms, sufficient to ensure an orderly transition of power via elections
+- Provide designated alternates in cases where quorum is required but not attainable with the current set of members
+- Communicate with the Continuous Delivery Foundation on a regular cadence
 
-The committee MUST define a contribution process that:
+The committee is responsible for a series of specific artifacts and activities:
 
-* Explains to potential contributors how/if they can add code to the repository/repositories
-* Documents Workflow and management of pull requests
-* Identifies who is authorized to commit or revert code
-* Identifies automation is required for normal operations
-* Defines how release decisions are made
-  * Who is authorized to release and when.
-  * Frequency limits
-* Defines the documentation process
-* Defines what Contributor License Agreement (CLA) process is required and how it is programmatically enforced before code is merged
+- The [Code of Conduct](https://github.com/cdevents/.github/blob/main/docs/CODE_OF_CONDUCT.md) and handling violations
+- The [Project Communication Channels](README.md#how-to-get-involved)
+- The [Contribution Process and Standards](README.md#contributing)
+- The [CDEvents Mission and Vision](https://cdevents.dev)
+- Select [election officers](#election-officers) to run elections
 
-### Security/Vulnerability Reporting and Response Process
+## Governance Meetings and Decision-Making Process
 
-* Identify and document where vulnerability reporting can be done to the project
-* Identify and document who is responsible for receiving vulnerability reports
-* Document process responsible parties go through to triage and determine veracity of vulnerability
-* Document process for facilitating fix, generating release update, and communicating vulnerability and fix to public
+Governance decisions, votes and questions should take place on the cdevents-governance@googlegroups.com mailing list.
+The mailing list is publicly readable by everyone.
+
+The Governing Board decisions are taken by seeking consensus towards a motion from all its members. If the consensus cannot be reached, the motion may be altered or dropped.
+
+## Elections
+
+### Voter Eligibility
+
+Anyone who has at least 5 [contributions](/processes.md#contributions) in the last 12 months. The [dashboard on cdevents.devstats.cd.foundation](https://cdevents.devstats.cd.foundation/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All) will show GitHub based contributions; contributions that are not GitHub based must be called out explicitly by the voter to confirm eligibility.
+
+### Candidate Eligibility
+
+Candidates themselves must be [contributors](/processes.md#contributor) who are eligible to vote, and must be nominated by contributors from at least two companies, which can include their own, and they can self-nominate.
```

Comment:
OK, I thought it would be clear from the statement below, but I can add it.
I think candidates are more important, so I would phrase it:

"Candidates may not act as election officers (see ##Election Officers below)"